tests/python/test_rank.py

```diff
@@ -0,0 +1,44 @@
+from xgboost.sklearn import XGBRanker
+import xgboost as xgb
+from xgboost import DMatrix
+import numpy as np
+from sklearn.datasets import load_svmlight_file
+
+train = load_svmlight_file('../../demo/rank/mq2008.train')
```

Comment:
This file needs to be downloaded by running `demo/rank/wgetdata.sh`.

Reply:
You are right. Actually I am not sure if this is the most appropriate way of testing it. Do you think if it is better to randomly generate a dataset and compare results using the generated dataset, or just add a fraction of the processed data to `demo/data`? Actually, after seeing many people ask about doing ranking task with xgboost in python, I am thinking about adding a demo for ranking, which probably also need the mq2008 data.